open-metadata-implementation/user-interfaces/presentation-server/ssl/keys/server.cert

```diff
@@ -0,0 +1,22 @@
+-----BEGIN CERTIFICATE-----
+MIIDnjCCAoYCCQDR7DKQq8tpSzANBgkqhkiG9w0BAQsFADCBkDESMBAGA1UECAwJ
+SGFtcHNoaXJlMRAwDgYDVQQHDAdIdXJzbGV5MRkwFwYDVQQKDBBMaW51eCBGb3Vu
```

Comment:
this is reintroducing a different key - we've just fixed this for 2.3 and are using the same key as the rest of the UI - for now. recommend this change is removed

Reply:
updated key to use one in master branch